manifests/0000_09_cluster-authentication-operator_05_deploy.yaml

```diff
@@ -1,27 +1,27 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  namespace: openshift-core-operators
-  name: origin-cluster-osin-operator
+  namespace: openshift-authentication-operator
+  name: origin-cluster-authentication-operator
   labels:
-    app: origin-cluster-osin-operator
+    app: origin-cluster-authentication-operator
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: origin-cluster-osin-operator
+      app: origin-cluster-authentication-operator
   template:
     metadata:
-      name: origin-cluster-osin-operator
+      name: origin-cluster-authentication-operator
       labels:
-        app: origin-cluster-osin-operator
+        app: origin-cluster-authentication-operator
     spec:
-      serviceAccountName: origin-cluster-osin-operator
+      serviceAccountName: origin-cluster-authentication-operator
       containers:
       - name: operator
         image: quay.io/openshift/origin-cluster-authentication-operator:latest
         imagePullPolicy: IfNotPresent
-        command: ["osin-operator", "operator"]
+        command: ["authentication-operator", "operator"]
```

Comment:
this is the only line I expect to change

Reply:
done

Reply:
so for both deployments, should be same, right? 
`command: ["authentication-operator", "operator"]`   (also in pkg/cmd/operator|operator2/cmd.go, same cmd.User for both? `operator`?)